Amazon.IonObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -652,7 +665,7 @@ public void DeserializeMultipleAnnotatedIonToClassNameWithSecondAnnotated()
             Assert.AreNotEqual(TestObjects.nativeTruck.ToString(), truck.ToString());
         }
 
-        private void AssertIsTruck(object actual)
+        private static void AssertIsTruck(object actual)
```

Comment:
This helper function should be moved to `Utils.cs` but I'm unsure if this is the right PR to make that change?

Reply:
Moved,